src/main/java/ludum/dare/actors/player/RunState.java

```diff
@@ -0,0 +1,30 @@
+package ludum.dare.actors.player;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+import ludum.dare.interfaces.IComponent;
+import ludum.dare.interfaces.IState;
+
+import java.util.Set;
+
+public class RunState extends IState {
+
+    public RunState(Set<IComponent> components) {
+        super(components);
+    }
+
+    public void enter() {
+        // TODO: switch to a running animation
+    }
+
+    public void exit() {
+
+    }
+
+    public IState update(float delta) {
+        // TODO: switch the direction of the animation based on something
+
+        if (!Gdx.input.isKeyPressed(Input.Keys.LEFT)) return new StandState(components);
```

Comment:
Since we're using something asides from keyboard controls, we're going to need to route this through something else. I'll try and rig up a controller component.